src/hub.js

```diff
@@ -739,13 +752,14 @@ document.addEventListener("DOMContentLoaded", async () => {
 
   const authChannel = new AuthChannel(store);
   const hubChannel = new HubChannel(store, hubId);
+  window.APP.hubChannel = hubChannel;
+
+  store.addEventListener("profilechanged", hubChannel.sendProfileUpdate.bind(hubChannel));
```

Comment:
Since HubChannel is passed a reference to store, this can just move into its constructor.